### PR TITLE
Support Keycloak.X in the conformance test-suite

### PR DESCRIPTION
**Pull request by thomasdarimont**
- Configuration of Keycloak variants extracted into dedicated docker-compose file
- Added dedicated configuration for keycloak-x container

Signed-off-by: Thomas Darimont <thomas.darimont@googlemail.com>

### CODE REVIEW

**Comment by VinodAnandan** [comment]
Thank you @thomasdarimont !!!

File: .env [hunk 1]
```diff
@@ -1,5 +1,5 @@
 OPENID_GIT_URL=https://gitlab.com/openid/conformance-suite.git
-OPENID_GIT_TAG=${OPENID_GIT_TAG:-release-v4.1.38}
+OPENID_GIT_TAG=${OPENID_GIT_TAG:-release-v4.1.39}
```
Comment (VinodAnandan) [inline .env:2]: Could you please test it with the latest conformance suite version ( release-v4.1.41) - https://gitlab.com/openid/conformance-suite/-/tags ?

Comment (thomasdarimont) [reply .env]: Thanks for the hint! Just updated the PR

Comment (thomasdarimont) [reply .env]: Thanks for the hint! Just updated the PR

**Comment by thomasdarimont** [comment]
The testrun fails with:
```
test_runner_1         | 
test_runner_1         | 
test_runner_1         | 2022-02-16 00:24:43: run-tests.sh starting
test_runner_1         | 
test_runner_1         | 
test_runner_1         | Run fapi1-advanced all tests
test_runner_1         | Traceback (most recent call last):
test_runner_1         |   File "/conformance-suite/scripts/run-test-plan.py", line 20, in <module>
test_runner_1         |     from conformance import Conformance
test_runner_1         |   File "/conformance-suite/scripts/conformance.py", line 12, in <module>
test_runner_1         |     import httpx
test_runner_1         | ModuleNotFoundError: No module named 'httpx'
test_runner_1         | 
test_runner_1         | 
test_runner_1         | 2022-02-16 00:24:43: run-tests.sh exiting
test_runner_1         | 
test_runner_1         | 
keycloak-fapi_test_runner_1 exited with code 123
```

**Comment by tnorimat** [comment]
@thomasdarimont I've been working on running FAPI1 Advanced, FAPI-CIBA and Open Banking Brazil FAPI1 conformance test with both WF based keycloak and Quarkus based keycloak based on your PR and I've found some issues.
I would like to tell you how to resolve these issues after resolving them.

**Comment by tnorimat** [comment]
@thomasdarimont @VinodAnandan I've found that conformance suite verson 4.1.40 and 41 cause the same problem @thomasdarimont has reported. Therefore, I would like to revert it to version 4.1.39 after merging this PR. I would like to treat this version matter afterwards.

**Comment by tnorimat** [comment]
@thomasdarimont Thank you!

**Pull request merged**